README.md

```diff
@@ -369,6 +388,30 @@ The `iotedgedev` module has the following commands:
 `iotedgedev project --help`
 - `--create TEXT`  Creates a new Azure IoT Edge project. Use `--create .` to create in current folder. Use `--create TEXT` to create in a subfolder.
 
+
+**azure**
+
+`iotedgedev azure --help`
+
+- `--setup`                    Reads the required Azure components
+                               configuration from your subscription. Creates
+                               new Azure resources or uses existing ones.
+                               [required]
+- `--azure-credentials TEXT TEXT`  The credentials (username password) to use to
+                               login to Azure. Leave empty to login in
+                               interactive mode.
+- `--subscription TEXT`        The Azure subscription name or id to use.
+                               [required]
+- `--resource-group-name TEXT` The name of the new Resource Group to use.
```

Comment:
Do you need "new" here?  That means a new one always created?  How about "The name of the Resource Group to use or create. Will be created if it doesn't exist.